### PR TITLE
Fix demodernizeShader to preserve parameter qualifiers

### PR DESCRIPTION
**Pull request by jjhembd**
# Description
This PR fixes a bug in the translation of voxel shaders to GLSL ES 1.0 for WebGL1 contexts.

If the user requests a webgl1 context, `demodernizeShader` replaces `in` variables like this:
```glsl
in vec2 v_textureCoordinates;
```

with the equivalent `varying` syntax in webgl1:
```glsl
varying vec2 v_textureCoordinates;
```

However,  some voxel shaders use `in` as a parameter qualifier in function signatures
```glsl
bool insideTile(in vec3 shapePosition, in ivec4 octreeCoords) {
```

The regex needed an edit to avoid substituting `varying` as a parameter qualifier.

## Testing plan

Load the [development voxel Sandcastle with a WebGL1 context](http://localhost:8080/Apps/Sandcastle/index.html#c=3Vt7U9tIEv8qKv6SQRGvpG4LQ+qIIbuug0DFXBIbU9mxNMa61cOrh2Ppyt/9eh4azUM2+LJ7G46isDXTv57unp7untHgJXGWW4sAf8OpdWbF+JvVw1lQRO4n2maPdzz63EviHAUxTsc7jvXvcWxZE5ThK1Ti9KSG9CP0iNOSNrrTNIlu02QR+Dg9z8rYswnIqmnvghBfo/kAp4vAwxxZ01PwP9OQQwRoUgShf534RYhp7855luE827/Dy7xIcbb/AcEnCi9Rms/6/fFOhzGgHx1Hkfo28H4jsk9RmGHa9YgTL/HVNg/UBuY38zwAS50wzS0rxb8XOMs/48ljeHhi5WnByFfwd9XpjuNxzGzqAhjHvs0VYI2fkiUO+3E2x16epNfBMogJhiMyD8fY9fGkeBzMkm/vUxTh7BanAwzC+DBJZDQ6xLSIPSIXSJ5fB/G7pIj9zM5maI47TNJgarFn6+zsTMwsGX5AWu/KOXYvr676t4Ob/kWn1s6jThHVHC2B7IFd4RuKj10vTGKsz4/GuUUsPh9d9imGcCsY5MA96NbmhWmMm27avCIqiz5oxlPwRx/6VoYx0PKPNkbN8buMoYmlGaPupcY4PKA/hklqoq1MAuvKwz5ZGYMgfgwxWXxUvHrBKXbKZ0HmckNZ9LMrmmWnaJleiVAyWIvqDaEfRDjOyOpS409jXvsnxyK/EigmawLo7yE8JWECQemh6czB3LSTc7rGOfJRjug0fLrsvZZovSSaw9zF+d0aUE8mcN9f3ZzfHR891BZmzpF5Kcq9WY9IoilBmmwWEJ6cBHeeJnlCpHd5fLkAAYChmEfqYgmLReOYTxeTIQduV3iBQ6DnFK5o66p0XzSaL1r/UOsfav0jrX/UZX5IFlgjx1tw4k4TMA0XFe7LGCt+oHmGNP6CWKwHrpQDWUPhLq1d+bFUHytZg5I6duMq9wcPUrc3Q3HcDNHmQ+DPwi0onU34dCQmzHMad3gfJig/PjpPU1Takgq7ynCdrmyQFMV+Eg0whpjPjMhsv0nPpbXHZ7C1mU10rRLKZy6kz490nA9FNCE5Bvt2M7Cs0qzAkjkINobcJoPtWv4p6G2HOLdoVO/Cx6k6GVa1tydcQ1CXjLpUqUtokagl+iWjX6r0S2hR6GsFQpzOcUp8f2ntW7Zinlfgqt12AFkMpQYoNwHI6qg0QFUDdMgMiIll94R4u5AH3wB5PTp5Phb9I/p8ZAxNFs2he6DhTLoFzbNvgN8RUO8SQzK2r0hzZw3K464sRzVao/2ShfbMsTLHWjhkfEeJhW0KBxABlrVDs5/qSZduaVoqvMV6u2f81XUF5AcPID5VA2Kr390CedggH1OM422wRw12EhZ4G+jxA52pv70RoBX7sjITPySQKMhI1siScIHte8H9gUzBqrumGLguwjx4YbXAa8civxJojnwfEuo7DIEBr8Udgn869aKVcefTXN/6bIT9AKWHYPE1JJm2TlVHDe+vKAzppPIKgijHko+OU3OOBjN5SQlBGVtn+70JlSc8OUfTPZxeCVxsWzWQ+K8h3WUr5bCFsmylHLVQVnouDHlxBhmLfT2VLMjbpLwliqLzvC7raNKdJ9/sI4eR1xlIl4cChKJc7V2J2SbcUMENn40bKbjREziyoeV61fHNkH/XFM1oGimsqVW479qy297TngdlJSjjNvKokZhv0ZpM1lZ96JLrFUhbhaOrplc57VWUob1SST2RZ1mmfdqmX6w9FVY+g2bZNWXQKi6DhVZEram7DIE3wkT1ZbjnGliK5xjRsPBmbe9wYy8Z8lDppf5xlXiILNumsONDGaSlSjoUpEODtFJJR4J0pJCKwfmXVyx4TMMEtqOsTbWFEKFsAZQtACFI1QKoWgDMYsuLYDqV5YKqs4Ws5GTlZrKKk1UqmUnoB1leR9Ds9zS3NeelfHb55x4ffpd/7vFxdtmnDG1TEYXzGWLluCaLCE8bq1TuL92tkIcNcrgd8qhBjrZD0hKVaivhVmvK1ZV6BNJeer74E5DnVkbGSYlUkD3/yOR/cZKhV9dmyW0S1yW1UWZ3tz7uaRpvGZ/W81eyFb0MMZDmGV/a2nZRkVhuoXK5S8dAlQaq1FGliaoMVKWjKr2QXVvkCQ/ptFrk5zCZ0BC8vUHk4qxN9c39ldKvqwPbwiAqIiZdD9KB3yri6PLjjYxCSxPVvifT5WEYdorjrOksN3XS05lGDZJnPSKBMHC7HBo1iWYbadXjQcF84yaLrtSGsXKOqGvDVwiZ3joIm53lps6q6dSL4Oed7QlG2xzxCdG2O+kT6rYd+DXz8cTiWDpW6ViV02DWnloxhmrC5xhXPcOSjN06P3vtTLYkX2qCyh7b9o4KRXOl7nlW2Kh/WNY0juFsSRxYRVqY7Tgmk6F+2qcwKQ0mZRsT7RS8UplUBpNKZyJZS6nnFDIzkqndRshyjInSh1jrXPXMreGgOZiIarttcXBvHZNt6VUPE8vRY8vRg+UohwloMTajdfy6b1aQUUd6D/o+tYmP9yIHPtzLdmph8owidPOJKZGu7cDUS2F/hW/TIAryAIjnvFqFmFFkeRINZog+RYmPQ8jfabDkVlDe6M9rBhkMGyULfB6GWl5YsGKY05EDlTUMwLn5+pRvbChoW8xDLe+J+Cb8VFbgRFWnJpG0OpEfOMGqI2cHLu9CueBAbz1cE6RrKqg0dCUeHopwimBDWd4l9LgYVvNgPsMptlWQO1F6ndoByT6DzN8JuVjAbmfUYnIvMAaX3unKtmh7sSt1M0tPU/RIAidrIzdSTqxfF0ngi55rFMT2e/7Qj+dFbk0z+ulADEjg0auir7WJcRqAn0f8C7WymNG6FWrm6bTIiCE5Jzfi5b3LX3Y8TroGSt2hgl1+ba6tmNp/ngU5/sG1X2Dv2AZ1Ov+NspJTK2844fk1TYkDD4XsesezUiYnugzDYJ6BCdzPPw9+eu3ydPER+UGROX8KKV2K45gpN0Cx76Esh4QI4eIuScIJSq9xXNj3kjlzOlXjHcEXcmezTbdITBzvcBHAWjiEJX0i7celeM/3gDzGcI95+upJE7afuJJTEzaTXI/YxBM9VjfMjdhnmWtc6pOcQh93Ja56GVbslSHJUukPZ8Te8Kr/4eLy4wuw4btk+cOZ793Nl5dquQznf5Dx1rwu/n+y3fEFs1mLxRC5w/pMu6FvKMhFsqAfxxeUsXrcqV9ztUAQ192H3wFs1kJMDgP2Ffw+ZcD+gsh1a84m2v1XlsTjnT/V3LQc+G5zb0g3f7nDvqSEI9vxRTmvEPwHcGGt5OtfXH64698N/5C8/5d78wvK/JIVX5Qv13K/lGj8IO9+ZrBNCPUbUAOPXHUbzJGHLxewtfmFEdnKeYSH4gXKCC/Og9wmpTu6czo1djNH5LiDbJGUF3aUi3bIIb++ogcA5NoZH408ym8MEtj73SZZQIc4s+ox3Dlvk19MkX/28OWDB36kAu22wqh5k5HAxilMHm0NSw+InHWGokvu6vL93dfeVb/3D0K84+ycZnkZ4rf1jPw9iOZJmlsFeBy4WI7BxWD6s/1JAUPlrpdl9aSd7svQUz9YWIF/1vJPOZYXoiyDnmkRQplbQcX89nQf6A1omCByWnKzwGmISkI2O3x7xRpd1z3dh8d2ZM72kBrn/wA). In the current main branch, shader compilation fails with many errors like this:

```javascript
RuntimeError: Fragment shader failed to compile.  Compile log: ERROR: 0:39: 'varying' : only allowed at global scope
ERROR: 0:39: 'varying' : Invalid parameter qualifier 
ERROR: 0:42: 'varying' : only allowed at global scope
ERROR: 0:42: 'varying' : Invalid parameter qualifier 
ERROR: 0:45: 'varying' : only allowed at global scope
ERROR: 0:45: 'varying' : Invalid parameter qualifier 
ERROR: 0:49: 'varying' : only allowed at global scope
ERROR: 0:49: 'varying' : Invalid parameter qualifier
```

The same shaders should compile successfully with this PR.

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [x] I have added my name to `CONTRIBUTORS.md`
- [x] I have updated `CHANGES.md` with a short summary of my change
- [ ] I have added or updated unit tests to ensure consistent code coverage
- [ ] I have update the inline documentation, and included code examples where relevant
- [x] I have performed a self-review of my code

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -34,6 +34,7 @@
 - Parts of the documentation have been updated to resolve potential issues with the generated TypedScript definitions. [#11776](https://github.com/CesiumGS/cesium/pull/11776)
 - Fixed type definition for `Camera.constrainedAxis`. [#11475](https://github.com/CesiumGS/cesium/issues/11475)
 - Fixed a geometry displacement on iOS devices that was caused by NaN value in `czm_translateRelativeToEye` function. [#7100](https://github.com/CesiumGS/cesium/issues/7100)
+- Fixed a bug affecting voxel shader compilation in WebGL1 contexts. [#11798](https://github.com/CesiumGS/cesium/pull/11798)
```
Comment (ggetz) [inline CHANGES.md:37]: This will now go into 115.

**Review by ggetz — COMMENTED** [review]
Looks good to me @jjhembd! Please merge in `main` and this should be good to go! Thanks!

**Comment by jjhembd** [comment]
Thanks @ggetz! I fixed the conflict in CHANGES.md

**Comment by ggetz** [comment]
Thanks @jjhembd!

**Pull request merged**